test/test.admin-myevents.selenium.js

```diff
@@ -0,0 +1,174 @@
+var expect = require('expect.js'),
+    common = require('./common'),
+    mandrill = require("mandrill-api"),
+    outbox = [];
+
+describe("ADMIN MYEVENTS SELENIUM", function() {
+    var browser = null,
+        event = null;
+
+    if (process.env.SKIP_SELENIUM_TESTS) {
+        return;
+    }
+    this.timeout(60000); // Extra long timeout for selenium :(
+
+    before(function(done) {
+        this.timeout(240000);
+
+        /* Mock Mandrill's API here */
+        mandrill.Mandrill = function(apiKey) {
+            this.messages = {
+                send: function(messageObj) {
+                    outbox.push(messageObj.message);
+                }
+            }
+        };
+
+        common.getSeleniumBrowser(function (theBrowser) {
+            browser = theBrowser;
+            common.standardSetup(done);
+        });
+    });
+
+    after(function(done) {
+        browser.quit().then(function() {
+            common.standardShutdown(done);
+        });
+    });
+
+    it("Can't access myevents page unauthenticated", function(done) {
+        browser.get(common.URL + "/myevents/");
+        browser.getCurrentUrl().then(function(url) {
+            expect(url).to.not.eql(common.URL + "/myevents/");
+            done();
+        });
+    });
+
+    it("Can't access myevents page if neither a superuser nor an admin of event", 
```

Comment:
as we are currently showing only those events which belong to an admin on myevents page, I am not sure how to add a test case that ensures that we don't let an admin proceed for an event they are not an admin of. Do you have any thoughts on this?